apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx

```diff
@@ -302,7 +302,7 @@ export const ReplaceConfirmationModal = ({ isOpen }: Props) => {
       dismissProps={{ outsidePressEvent: 'mousedown' }}
     >
       <div className="flex h-full flex-col justify-between gap-2 sm:gap-4">
-        <div className="flex min-h-[80%] flex-col gap-4 md:p-6">
+        <div className="flex min-h-[80%] flex-col gap-4 px-3 py-4 md:p-6">
```

Comment:
This modal window will be changed in scope of #1230. Please, do not do any changes here to avoid merge conflicts.

Reply:
ok thanks, I'll change it now